### PR TITLE
AD trust installer modularization: prelude

### PR DESCRIPTION
**Pull request by martbab**
This PR is more of a preparatory work for modularization of AD trust installer
code. The code was formatted to conform with PEP-8, explicit exit statements
were replaced with exceptions and a minor pylint issues were fixed.

### CODE REVIEW

**Comment by abbra** [comment]
LGTM -- I haven't run the code but read through it.

File: install/tools/ipa-adtrust-install [hunk 1]
```diff
@@ -232,35 +254,40 @@ def main():
...
+        "missing options might be asked for interactively later\n")
     root_logger.debug('IPA version %s' % version.VENDOR_VERSION)
 
     check_server_configuration()
 
     fstore = sysrestore.FileStore(paths.SYSRESTORE)
 
-    print("==============================================================================")
-    print("This program will setup components needed to establish trust to AD domains for")
+    print("================================================================"
```
Comment (MartinBasti) [inline install/tools/ipa-adtrust-install]: Pro tip: print("=" * 80)

Comment (MartinBasti) [inline install/tools/ipa-adtrust-install]: 78 actually

**Comment by martbab** [comment]
Fixed upstream
master:
https://fedorahosted.org/freeipa/changeset/847be3a8a85cd58e5a011c0c2bc7e1123eb4a1aa
https://fedorahosted.org/freeipa/changeset/e27f6bfdc31b767be9ded411e869716b76f478ce
https://fedorahosted.org/freeipa/changeset/d7cfbb870fce40b50f6df2446c864099f8ea833e

**Pull request closed without merging**